test/controllers/foo_controller_test.rb

```diff
@@ -0,0 +1,3 @@
+begin 
+rescue Exception
```

Comment:
__[rubocop]__
* Lint/HandleExceptions: Do not suppress exceptions. [:ref](https://github.com/bbatsov/ruby-style-guide#dont-hide-exceptions)
* Lint/RescueException: Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`? [:ref](https://github.com/bbatsov/ruby-style-guide#no-blind-rescues)